### ISSUE TITLE
Ghosting effect with black background

### ISSUE THREAD

**Issue by pmsobrado** [skyfloogle/red-viper#34]
I've been noticing some ghost effect on the upper screen when using Red Viper. At first I thought my 3DS was broken or something. Then I learned that this is normal when the image has contrast, like in this case, a 100% black background. The image of one eye slightly bleeds into the other eye.

Maybe the background color could be changed to something less dark?

Again, this is not a Red Viper issue at all, just asking if this is possible or makes sense.

Thanks.

**Comment by victorkte** [comment]
I also noticed that this ghosting issue is not present on the first release of red viper, at least for me... and on the newest release if you tweak the 3D slider you can get the ghosting to be less noticeable, perhaps this is something from the new 3D slider update?

**Comment by pmsobrado** [comment]
True, is more noticeable after the 3D slider support. But even with the 3D effect as before, or by default, it is noticeable in some parts depending of the game. I guess this is how the 3D works and the problem with contrasts, nothing we can do and nothing broken with the emulator, but if changing the background color is not difficult, maybe it could palliate the effect.

**Comment by Pat86** [comment]
Would be nice to know the slider position for 3d depth that equals the Virtual Boy.

**Comment by pmsobrado** [comment]
That's resolved here: https://github.com/skyfloogle/red-viper/issues/2

**Comment by djedditt** [comment]
> Would be nice to know the slider position for 3d depth that equals the Virtual Boy.

@Pat86 The Virtual Boy itself needs to be adjusted per user with the IPD dial and the focus slider. The default setting is when the squares in the corners line up vertically in the IPD adjustment screen, which is shown at the start of every game. In Nintendo 3DS mode this means the slider would have to be at the bottom for the squares to line up. In Virtual Boy IPD mode the slider would have to be in the middle. It's best to verify visually, that's what the screen is for:

![image](https://github.com/skyfloogle/red-viper/assets/10092195/667d8a89-cc17-49f4-966f-12551ea303a7)

Note: If you are playing a game, press START to pause the game, then press SELECT to get to this screen. Pressing START or SELECT again will exit the IPD adjustment screen.

**Comment by skyfloogle** [comment]
The 3DS's display is not perfect, and may display weird ghosting-type artifacts under certain circumstances. As of 4a88fe09de99963936c83ef3ae0f7023c9a01091, more flexible palette options are available, which may help to alleviate these issues if you so desire. I'm not planning on changing the default colour scheme from red/back though - it is the Virtual Boy, after all.
